diff --git a/main.patch b/main.patch
index 3df4ff3..e69de29 100644
--- a/main.patch
+++ b/main.patch
@@ -1,11 +0,0 @@
-diff --git a/src/main.c b/src/main.c
-index b5e50a1..334910f 100644
---- a/src/main.c
-+++ b/src/main.c
-@@ -2,5 +2,6 @@
- 
- int main(void){
- 	printf("Hello ESPL\n");
-+	printf("This is a new print\n");
- return 0;
- }
diff --git a/src/main.c b/src/main.c
index 3e7093c..1c1db4c 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1,12 +1,18 @@
 #include <stdio.h>
-#include<espl_lib.h>
+#include <espl_lib.h>
 
 int main(void){
 	printf("Hello ESPL\n");
-	printf("This is a new print\n");
-	unsigned int input = getchar();
-	char output;
-	output =  num_to_words(input);
-	printf("%c\n", output);
+	char stop = '1';
+	do {
+		printf("\nPlease enter Number: ");
+		unsigned int input;
+		scanf("%d", &input);
+		char *output =  num_to_words(input);
+		printf("%c\n", *output);
+		printf("Wish to exit? Press 0. Else press any other key: ");
+		while ((getchar()) != '\n');
+		stop = getchar(); //flushes the input buffer
+	} while (stop != '0');
 return 0;
 }
